css/style.css

```diff
@@ -0,0 +1,264 @@
+@import url('https://fonts.googleapis.com/css2?family=Playfair+Display:wght@900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=PT+Sans+Narrow:wght@700&family=Playfair+Display:wght@900&display=swap');
+.content{}
+header{
+  height: 660px;
+  background: rgba(158, 140, 108, 0.2);
+  color: #fff;
+  position: relative; 
+  /* padding: 0px; */
+  display: flex;
+  padding-top: 70px;
+  align-items: flex-start;
+  justify-content: center;
+}
+
+header::before,
+header::after {
+  content: '';
+  position: absolute;
+  height: 100px;
+  bottom: -55px;
+  right: 0;
+  left: 0;
+  background: #fff;
+  transform: skewY(-3deg);
+  -webkit-transform: skewY(-6deg);
+  -moz-transform: skewY(-3deg);
+  -ms-transform: skewY(-3deg);
+}
+
+.head .head-upper{
+  margin-top: 0px;
+  padding: 0px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.head .head-upper > div >div{
```

Comment:
The simpler the selector, the faster and easier it is for the browser to render your website. Even though this one has almost insignificant performance benefits, it would be great to use a class for the div, rather than select it using `div > div > div`. Also, for someone reading your css, it should be easy to find the element in the html. For that, its better to use a class.